<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_vitesseDefil</class>
 <widget class="QDialog" name="Dialog_vitesseDefil">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>295</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="l_vitesse">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Choissez le vitesse de défilement du Diaporama :</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="GB_choix">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Vitesse de défilement</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="rb_vitesseX1">
        <property name="text">
         <string>x1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rb_vitesseX2">
        <property name="text">
         <string>x2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rb_vitesseX5">
        <property name="text">
         <string>x5</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rb_vitesseX10">
        <property name="text">
         <string>x10</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="HL_buttons">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="b_ok">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="b_quitter">
       <property name="text">
        <string>Quitter</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
